LondonBritishLibrary/orient/BLorient12859.xml

```diff
@@ -0,0 +1,437 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="BLorient12859">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Magic scroll</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>BL Oriental 12859</idno>
+                        <altIdentifier><idno>Or. 12859</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 80</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title ref="LIT5888AsmatPrayer">Prayer for gaining respect</title>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ሀሊም፡ ወከመ፡ ቢስርቀያድ፡ በሥልጣነ፡ ፲ወ፬አስማተ፡ ርብሽመ፡ ከማሁ፡ ሙሮሸብአ፡
+                                ለሰብአ፡ ጋላ፡ ወለሻንቅላ፡ ወለሰብአ፡ ኵሉ፡ ፍጥረት፡ በግርማ፡ ገጸ፡ ገብርከ፡ <gap reason="lost"/></incipit>
+                            <note>Written in the left column under the third picture.</note>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <title ref="LIT2393tabiba"/>
+                            <note>First stanza of the hymn. Written in the left column under the third picture.</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> እግዚአብሔር፡ ጠቢበ፡ ጠቢባን፡ ከሃሊ፤ እስመ፡ ለዓለመ፡ ዓለም፡ አሜን፡ እምጥንት፡ አንተ፡
+                                ኢትበሊ፤ ድኩመ፡ ሕሊና፡ ገብርከ፡ <gap reason="lost"/></incipit>
```

Comment:
If I understand Strelcyn and your description, the name of the owner has systematically been erased. Then maybe one could encode it `<del rend="effaced"/>`?